### ISSUE TITLE
Strange behaviour on the glossary term relations: inherits and contains

### ISSUE THREAD

**Issue by devantler** [datahub-project/datahub#6280]
**Describe the bug**

The inherited relation on glossary terms seems to be reversed. When adding inheritance from one glossary term to another, the parent of those two has the children's entities in its related entities. I would expect the child glossary term to inherit all the related entities from its parent.

The contains relation does not seem to do anything.

**To Reproduce**
Steps to reproduce the behaviour:
1. Add two Glossary Terms
2. Let one Glossary Term inherit from another one.
3. Add the Glossary Terms to some entities.
4. Observe that the inherited relation is reversed.

**Expected behaviour**

I expect the child to inherit the parent's entities, as this is the expected behaviour of normal inheritance. Am I mistaken in this, or is something afoot?

I expect the contain relation to add the contained glossary term entities to the glossary term owning the relation.

**Desktop (please complete the following information):**
 - OS: macOS (datahub docker quickstart)
 - Browser: Safari
 - Version 0.9.0

**Comment by aditya-radhakrishnan** [comment]
Hi @devantler I'm a little confused by the issue. Glossary terms can only be children of Glossary nodes, not Terms. I'm not sure how you're observing that the inherited relation is reversed. Could you please given some more context and maybe post some screenshots? Thank you!

**Comment by devantler** [comment]
@aditya-radhakrishnan I am very occupied with work and my thesis in the coming time, so I will not be able to help further with this issue. But if it is not reproducible, I believe it might have been fixed in the meantime. It was very obvious when we discovered it in class, so I think you would have noticed the issue if it was still an issue.

Feel free to close the issue. When I need DataHub again, I will make sure to check if it is still an issue and reopen this issue if it is :-) 

Reminder to @devantler to check up on this.
